go.sum

```diff
@@ -1330,6 +1330,8 @@ github.com/joho/godotenv v1.3.0/go.mod h1:7hK45KPybAkOC6peb+G5yklZfMxEjkZhHbwpqx
 github.com/jonboulle/clockwork v0.1.0/go.mod h1:Ii8DK3G1RaLaWxj9trq07+26W01tbo22gdxWY5EU2bo=
 github.com/jonboulle/clockwork v0.2.2 h1:UOGuzwb1PwsrDAObMuhUnj0p5ULPj8V/xJ7Kx9qUBdQ=
 github.com/jonboulle/clockwork v0.2.2/go.mod h1:Pkfl5aHPm1nk2H9h0bjmnJD/BcgbGXUBGnn1kMkgxc8=
+github.com/josh-diamond/shepherd v0.0.0-20240214234402-e569c4ded1bd h1:OOZvtalt6w6kIZuh2aoZJ88BFZ+7cNr3hlzNzR30oVk=
```

Comment:
This will be addressed once you run `go mod tidy` after you have the official repo back in place.